packages/beacon-node/src/network/reqresp/utils/dataColumnResponseValidaiton.ts

```diff
@@ -29,7 +29,9 @@ export async function handleColumnSidecarUnavailability({
     availableColumns: availableColumns.join(","),
     slot,
   };
-  if (blockRoot) logData.blockRoot = toRootHex(blockRoot);
+  if (blockRoot) {
+    logData.blockRoot = toRootHex(blockRoot);
```

Comment:
Should pretty print the blockRoot i think

Reply:
I don't think there is nay such helper. We had been using `toRootHex` everywhere for the logs.

Reply:
`import {prettyBytes} from "@lodestar/utils";`

Reply:
it prints `0x12...3ab6`